dotnet/src/webdriver/WebDriver.cs

```diff
@@ -378,6 +378,7 @@ public PrintDocument Print(PrintOptions printOptions)
         {
             if (printOptions is null)
             {
+                // TODO the interface names this parameter 'options', but we will be throwing for 'printOptions'. Confusion for users?
```

Comment:
No please, todo comment in "legacy" will be never resolved. Manage it via issue tracker.

Reply:
Opened https://github.com/SeleniumHQ/selenium/issues/14784

```suggestion
```